README.md

```diff
@@ -1,4 +1,8 @@
-# Scaleway Terraform Provider
+<a href="https://terraform.io">
+    <img src="https://www.datocms-assets.com/2885/1629941242-logo-terraform-main.svg" alt="Terraform logo" title="Terraform" vertical-align="right" height="50" />
```

Comment:
This is not an hashicorp domain nor a github domain. I prefer having no logo than a logo hosted on a place that is not official.

Reply:
I've took the url from the official terraform [repository](https://raw.githubusercontent.com/hashicorp/terraform/main/README.md). Thought, I can take the file.